src/components/views/spaces/threads-activity-centre/ThreadsActivityCentre.tsx

```diff
@@ -84,6 +85,11 @@ export function ThreadsActivityCentre({ displayButtonLabel }: ThreadsActivityCen
                     <ThreadsActivityCentreButton
                         displayLabel={displayButtonLabel}
                         notificationLevel={roomsAndNotifications.greatestNotificationLevel}
+                        onClick={async () => {
+                            // Open the TAC after the release announcement closing
+                            setOpen(true);
+                            await ReleaseAnnouncementStore.instance.nextReleaseAnnouncement();
```

Comment:
What if the current announcement is not TAC-related?

Reply:
If the current announcement is not TAC-related, the TAC release announcement is not displayed.
Only one release announcement can be displayed at one time. See https://github.com/matrix-org/matrix-react-sdk/pull/12380

Reply:
Ah, it wasn't obvious this is in an `isReleaseAnnouncementOpen` scope